src/elements/kw-projects-header/kw-projects-header.html

```diff
@@ -100,33 +101,32 @@
                 width: 60px;
             }
             .step-icon a {
-                /*a to take icon's dimension and disable link by default*/
                 display: flex;
                 flex: 1;
                 height: 100%;
+                /*disabling link by default*/
                 pointer-events: none;
             }
+            .step-icon.current a,
+            .step-icon.done a,
+            kano-step:first-child .step-icon a {
+                pointer-events: auto;
+            }
+            .step-icon.locked {
+                background-image: url("/assets/projects/simple-progress/lock-circle.svg");
+            }
             kano-step:first-child .step-icon:not(.done) {
                 background-image: url("/assets/projects/simple-progress/make-button.svg");
                 background-size: 82%;
                 width: 110px;
                 height: 60px;
             }
-            .step-icon.locked {
-                background-image: url("/assets/projects/simple-progress/lock-circle.svg");
-            }
             .step-icon.current {
                 background-image: url("/assets/projects/simple-progress/make-button.svg");
                 background-size: 82%;
```

Comment:
Could the `background-size`, `width` and `height` just be applied to `step-icon` to avoid repeating, and then just updating the `background-image` with the more specific selectors?